scripts/hid_configurator/NrfHidDevice.py

```diff
@@ -485,3 +485,9 @@ def config_get(self, module_name, option_name, poll_interval=POLL_INTERVAL_DEFAU
 
     def config_set(self, module_name, option_name, value, poll_interval=POLL_INTERVAL_DEFAULT):
         return self._config_operation(module_name, option_name, False, value, poll_interval)
+
+    def get_module_variant_name_by_root(self, name):
```

Comment:
Maybe call it: `get_module_variant(self, name)`? That would be much simpler.

I would also prefer for this function to return whole name used to identify the module to avoid using information about naming convention in other DFU related scripts if it's not necessary (we keep information in a single place). So that we may call it e.g. `get_complete_module_name`?